### PR TITLE
Added some hints to run the plugin.

### PR DESCRIPTION
**Pull request by Danie**
Possibly helpful for less experienced Moodle users or newbies (as I am).

### CODE REVIEW

**Review by berndwa — APPROVED** [review]

**Comment by berndwa** [comment]
Should be updated to work with the newest Moodle-Version 3.10.*

File: README.md [hunk 1]
```diff
@@ -28,7 +28,12 @@ Optionnally you can:
 * change the other plugin settings as required,
 * tweak the execution times of the scheduled task running the file conversions.
 
-## Issues
+Hints:
+
+* Read https://docs.moodle.org/310/en/Managing_filters how to activate and order filters.
```
Comment (ndunand) [inline README.md]: @Danie could you please change this URL to https://docs.moodle.org/en/Managing_filters to be version agnostic?

**Comment by ndunand** [comment]
great idea, thanks!

File: README.md (hunk 1 above)
Comment (Danie) [reply README.md]: done

**Comment by ndunand** [comment]
Thanks @Danie this is now merged and will be in the next release.

Hopefully it will help in avoiding some headaches

**Pull request merged**